cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Settings page', () => {
-  before(() => {
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import SettinsPageObject from '../support/pages/settingPageObjects';
+import homePageObject from '../support/pages/home.pageObject';
 
-  });
+const homePage = new homePageObject();
+const signInPage = new SignInPageObject();
+const settingsPage = new SettinsPageObject();
 
-  beforeEach(() => {
+const userData = {
+  username: 'riot',
+  email: 'riot@qa.team',
+  password: '12345Qwert!',
+};
 
+describe('Settings page', () => {
+  let user;
+
+  beforeEach(() => {
+    cy.task('db:clear');
+    homePage.visit();
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
+    settingsPage.loginUser();
+    settingsPage.visit();
   });
 
   it('should provide an ability to update username', () => {
-
+    settingsPage.updateUsername(user.username);
+    settingsPage.clickSubmitButton();
+    homePage.usernameLink
+      .should('contain', user.username);
   });
 
   it('should provide an ability to update bio', () => {
-
+    settingsPage.updateBio(user.bio);
+    settingsPage.clickSubmitButton();
+    homePage.clickSettingsLink();
+    settingsPage.assertBio(user.bio);
   });
 
   it('should provide an ability to update an email', () => {
-
+    settingsPage.clickSubmitButton();
```

Comment:
The 'clickSubmitButton' method is called before updating the email. It should be called after updating the email.